helm/nessie/values.yaml

```diff
@@ -0,0 +1,79 @@
+replicaCount: 1
```

Comment:
there is no config to set up tracing here?

Reply:
the idea was to create an MVP version to be able to install Nessie via Helm and later add/parameterize what we need

Reply:
does that sound reasonable @rymurr?

Reply:
tracing should be a 2 liner and is part of the MVP imho. it should be easy for a user to:
* start nessie 
* secure it
* attach it to a valid ingress
* observe it
We've done everything but the tracing part of observing

Reply:
added tracing stuff